xbmc/cores/VideoPlayer/DVDCodecs/Overlay/OverlayCodecWebVTT.cpp

```diff
@@ -38,11 +39,21 @@ bool COverlayCodecWebVTT::Open(CDVDStreamInfo& hints, CDVDCodecOptions& options)
   if (!m_webvttHandler.Initialize())
     return false;
 
-  // "fmp4" extradata is provided by InputStream Adaptive to identify
-  // WebVTT in MP4 encapsulated subtitles (ISO/IEC 14496-30:2014)
-  if (hints.extradata && std::strncmp(static_cast<const char*>(hints.extradata), "fmp4", 4) == 0)
+  // Extradata provided by InputStream Adaptive
```

Comment:
Since ISA is an add-on I'd rewrite the comment to be more generic, eg:
Extradata can be provided by Inputstream addons (e.g. inputstream.adaptive)